src/index.js

```diff
@@ -20,9 +20,21 @@ function docsifyCopyCode(hook, vm) {
                 button.style.background = vm.config.themeColor;
             }
 
-            button.addEventListener('click', function(event) {
+            element.appendChild(button);
+        });
+    });
+
+    hook.mounted(function() {
+        const listenerHost = document.querySelector('.content');
+
+        listenerHost.addEventListener('click', function(evt) {
```

Comment:
Very nice adding the event handler to the containing element making it more resilient to changes afterwards.

Reply:
Makes total sense. Nice find 👏